NewsApp/NewsApp/Controller/SearchController.swift

```diff
@@ -9,40 +9,182 @@
 import Foundation
 import UIKit
 
-class SearchController: UIViewController {
+class SearchController: UITableViewController {
     @IBOutlet private weak var searchBar: UISearchBar!
+    @IBOutlet private weak var sortByLabel: UILabel!
+    @IBOutlet private weak var sortBySegmentedControl: UISegmentedControl!
+    @IBOutlet private weak var dateFromLabelWithTitle: UILabel!
+    @IBOutlet private weak var dateFromLabelWithDate: UILabel!
+    @IBOutlet private weak var dateToLabelWithTitle: UILabel!
+    @IBOutlet private weak var dateToLabelWithDate: UILabel!
+    @IBOutlet private weak var languageLabelWithTitle: UILabel!
+    @IBOutlet private weak var languageLabelWithLanguage: UILabel!
+    @IBOutlet private weak var searchButton: UIButton!
+    @IBOutlet private weak var moreFiltersButton: UIButton!
+    @IBOutlet private weak var moreFiltersImage: UIImageView!
+
+    var parametersRequest: ParametersRequest = ParametersRequest(requestType: .everything)
+    private var showMoreFilters: Bool = false
+    private let sortBy = [SortBy(title: R.string.localizable.publicationDate(), sortByType: .publishedAt),
+                          SortBy(title: R.string.localizable.popularity(), sortByType: .popularity),
+                          SortBy(title: R.string.localizable.relevancy(), sortByType: .relevancy)]
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        showOrNotMoreFilters(show: false)
         initSearchController()
     }
 
-    func initSearchController() {
-        self.navigationItem.title = R.string.localizable.search()
-        self.searchBar.placeholder = R.string.localizable.searchForAnArticle()
+    override func viewWillAppear(_ animated: Bool) {
+        initFilters()
+    }
+
+    @IBAction func onSearchButtonClicked() {
+        search()
+    }
+
+    @IBAction func onMoreFiltersButtonClicked() {
+        showOrNotMoreFilters(show: !showMoreFilters)
+        switch showMoreFilters {
+        case true:
+            moreFiltersButton.setImage(UIImage(systemName: Constantes.SystemImage.chevronUp), for: .normal)
+        case false:
+            moreFiltersButton.setImage(UIImage(systemName: Constantes.SystemImage.chevronDown), for: .normal)
+        }
```

Comment:
Pas besoin d'utiliser le switch pour les boolean

Reply:
fix in "fix PR filtres recherche"